src/queue.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+export default class Queue {
+  constructor(initialValues=[]) {
```

Comment:
Indent this a level. It is convention for every method within in a class to have one level of indentation.